src/collective/blueline/browser/viewlets.py

```diff
@@ -31,6 +34,25 @@ def show(self):
         show = api.user.is_anonymous() or self.show_authenticated
         return not is_configlet and show
 
+    def evaluate_code(self, code):
```

Comment:
@hvelarde I tried to access chameleon classes but there is no simple way to do it.

I think we should change plone classes to accept a string instead of a file when process page template, this could be a better way to enhance this method.